examples/gnrc_networking/Makefile

```
@@ -27,7 +32,16 @@ USEMODULE += shell_cmds_default
 USEMODULE += ps
 USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
+
+# Optionally include the RPL routing protocol (particular useful if not all
+# nodes are within radio transmission range to the border router)
+USE_RPL ?= 0
```

```suggestion
USE_RPL ?= 1
```

I see no reason (except that the border router is not automatically initialized as RPL root) to make this opt-in.

---

Hm, how often have you (or anyone else) used RPL with this example?

The biggest arguments for disabling RPL here for meare reduced complexity and reduced code size.
gnrc_coap and gnrc_networking_mac (for some reason) do include RPL.

---

> Hm, how often have you (or anyone else) used RPL with this example?

That's exactly my point. Everyone who uses this example currently uses RPL to a degree. Without it, the user base would shrink significantly, which would increase bit rot.

---

Not sure to which degree RPL is used in these scenarios (apart from trickle timers and the like). But I sense that we need a more general discussion with a broader range of the community about the purpose of this (and the other example applications) first. Currently, I would prefer this application as a simple host application and rather introduce a dedicated RPL example application.